vendor/github.com/coreos/go-semver/NOTICE

```diff
@@ -0,0 +1,5 @@
+CoreOS Project
```

Comment:
Why do we still have vendor in here?

Reply:
Good point. I guess we can delete it now.